### PR TITLE
Azure lib changes in account and bucket servers - Part 2/4

### PR DESCRIPTION
**Pull request by romayalon**
Signed-off-by: Romy <romy2232@gmail.com>

### Explain the changes
1. Applied the new azure library functions in account server and bucket server (also added here changes to new azure wrap, package.json, and package_lock.json that will be removed after merging the first PR - https://github.com/noobaa/noobaa-core/pull/6744)

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1.

### CODE REVIEW

**Review by liranmauda — COMMENTED** [review]
Package.json:
looks like some module got back in... 
Need to rebase and see that those modules are not here nad have the correct versions

File: src/server/system_services/bucket_server.js [hunk 1]
```diff
@@ -1103,15 +1103,17 @@ function get_cloud_buckets(req) {
                 req.rpc_params.connection
             );
             if (connection.endpoint_type === 'AZURE') {
-                let blob_svc = azure_storage.createBlobService(cloud_utils.get_azure_connection_string(connection));
+                let blob_svc = azure_storage.BlobServiceClient.fromConnectionString(
```
Comment (liranmauda) [inline src/server/system_services/bucket_server.js:1106]: const

File: src/server/system_services/bucket_server.js [hunk 2]
```diff
@@ -1103,15 +1103,17 @@ function get_cloud_buckets(req) {
...
-                return P.timeout(EXTERNAL_BUCKET_LIST_TO, P.fromCallback(callback => {
-                        blob_svc.listContainersSegmented(null, { maxResults: 100 }, callback);
-                    }))
-                    .then(data => data.entries.map(entry =>
-                        _inject_usage_to_cloud_bucket(entry.name, connection.endpoint, used_cloud_buckets)
-                    ));
+                return P.timeout(EXTERNAL_BUCKET_LIST_TO,
+                        blob_svc.listContainers().byPage({ maxPageSize: 100 }).next())
+                    .then(iterator => {
+                        let response = iterator.value;
```
Comment (liranmauda) [inline src/server/system_services/bucket_server.js:1113]: const

**Review by liranmauda — COMMENTED** [review]
We need to verify that after the merge from part 1/4 we get the code we intended 🙂

**Comment by romayalon** [comment]
@liranmauda after rebasing the branch with PR 1/4 merged to master, I was able to remove the duplicate files.

**Review by liranmauda — APPROVED** [review]
LGTM

**Pull request merged**